Feature/docs (#273)
infinum/eightshift-docs: website/forms/features/cloudflare.md

title: Cloudflare
---

## What is Cloudflare?

> Cloudflare is a large network of servers that can improve the security, performance, and reliability of anything connected to the Internet. Cloudflare does this by serving as a reverse proxy Open external link for your web traffic.

Reviewer: ```suggestion
Cloudflare is a large network of servers that can improve the security, performance, and reliability of anything connected to the Internet. This is done by serving content as a _reverse proxy_ for all of your website's traffic.
```

![Cloudflare screen](/img/forms/cloudflare.webp)

> ## How do we use Cloudflare?

Reviewer: ```suggestion
## How to use the Cloudflare integration?
```

Author: this is not integration so this title is not correct

Reviewer: updated!

> Eightshift Forms doesn't configure the Cloudflare app or any other third-party tools. However, enabling this feature adds necessary configurations in the backend for everything to function correctly.

Reviewer: ```suggestion
After setting up Cloudflare, head to the Global settings and enable the Cloudflare integration. This will set up all the necessary options in the backend for everything to function properly.
```

Author: no an integration

Reviewer: updated!

## Features affected by Cloudflare

Cloudflare affects the following features:

### Geolocation

> By default geolocation uses IP database to get users location. With Cloudflare enabled, geolocation will use Cloudflare headers to get the user location.

Reviewer: ```suggestion
By default geolocation uses the IP database to get user's location. With Cloudflare enabled, geolocation will use Cloudflare headers as a data source.
```

> Make sure you have enabled IP Geolocation in your Cloudflare dashboard. You can find more details on how to enable it [here](https://developers.cloudflare.com/support/network/configuring-ip-geolocation/).

Reviewer: ```suggestion
:::note
Make sure IP geolocation is enabled in the Cloudflare dashboard. More details on the Cloudflare IP geolocation functionality can be found [here](https://developers.cloudflare.com/support/network/configuring-ip-geolocation/).
:::
```

### Security

> When using Cloudflare, the user's IP address is masked and replaced with the IP address of the Cloudflare server. This ensures proper functionality.

Reviewer: ```suggestion
When using Cloudflare, the user's IP address is masked and replaced with the IP address of the Cloudflare server. This ensures everything keeps working fine, with keeping all of the users private.
```
